### PR TITLE
read keys from multiple config stores

### PR DESCRIPTION
**Pull request by sophiaso**

### CODE REVIEW

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java [hunk 1]
```diff
@@ -34,15 +34,14 @@
     private ScheduledFuture<?> watchFuture;
     private AzureCloudConfigProperties properties;
     private boolean firstTime = true;
-    // TODO (wp) multi stores is not supported yet
-    private ConfigStore configStore;
+    private List<ConfigStore> configStores;
```
Comment (Incarnation-p-lee) [inline spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java]: not sure final is needed or not here.

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java [hunk 2]
```diff
@@ -88,33 +87,37 @@ public int getPhase() {
...
+                    firstTime = false;
+                    return;
+                }
+
+                Optional<String> changedKey = newKeyEtagMap.entrySet().stream()
+                        .filter(e -> !mapInclude(keyNameEtagMap, e.getKey(), e.getValue()))
+                        .map(e -> e.getKey())
+                        .findFirst();
+
+                if (!firstTime && changedKey.isPresent()) {
```
Comment (Incarnation-p-lee) [inline spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java]: seems firstTime is always false here ?

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java [hunk 3]
```diff
@@ -15,17 +15,13 @@
 
 public class AzureConfigPropertySource extends EnumerablePropertySource<ConfigServiceOperations> {
     private final String context;
-    private final AzureCloudConfigProperties configProperties;
     private Map<String, Object> properties = new LinkedHashMap<>();
-    // TODO (wp) multi stores is not supported yet
     private ConfigStore configStore;
```
Comment (Incarnation-p-lee) [inline spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java]: final ?

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java (hunk 1 above)
Comment (sophiaso) [reply spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java]: Thanks, added final.

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java (hunk 2 above)
Comment (sophiaso) [reply spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java]: Removed redundant condition check.

File: spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java (hunk 3 above)
Comment (sophiaso) [reply spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java]: Updated.

**Pull request merged**